service/test/_registry_test.js

```diff
@@ -680,6 +680,114 @@ describe('Registry', function () {
+        }, ReferenceError);
+      });
+    });
+
+    testErrorCallback('importDevicesFromBlobByIdentity', 'input', 'output');
+
+    /*Tests_SRS_NODE_IOTHUB_REGISTRY_16_031: [The `importDeviceFromBlob` method shall construct an HTTP request using information supplied by the caller, as follows:
+    ```
+    POST /jobs/create?api-version=<version> HTTP/1.1
+    Authorization: <config.sharedAccessSignature>
+    Content-Type: application/json; charset=utf-8
+    Request-Id: <guid>
+
+    {
+      "type": "import",
+      "inputBlobContainerUri": "<input container Uri given as parameter>",
+      "outputBlobContainerUri": "<output container Uri given as parameter>"
+    }
+    ```]*/
+    it('constructs a valid HTTP request', function (testCallback) {
+      var fakeInputBlob = "input";
+      var fakeOutputBlob = "output";
+      var fakeHttpHelper = {
+        executeApiCall: function (method, path, httpHeaders, body, done) {
+          assert.equal(method, 'POST');
+          assert.equal(path, '/jobs/create' + endpoint.versionQueryString());
+          assert.equal(httpHeaders['Content-Type'], 'application/json; charset=utf-8');
+          assert.equal(body.type, 'import');
+          assert.equal(body.inputBlobContainerUri, fakeInputBlob);
+          assert.equal(body.outputBlobContainerUri, fakeOutputBlob);
+          assert.equal(body.storageAuthenticationType, "IdentityBased");
+          done();
+        }
+      };
+
+      var registry = new Registry(fakeConfig, fakeHttpHelper);
+      registry.importDevicesFromBlobByIdentity(fakeInputBlob, fakeOutputBlob, testCallback);
+    });
+  });
+
+  describe('#exportDevicesToBlobByIdentity', function () {
+    /*Tests_SRS_NODE_IOTHUB_REGISTRY_16_004: [A `ReferenceError` shall be thrown if outputBlobContainerUri is falsy]*/
+    [undefined, null, ''].forEach(function(badUri) {
+      it('throws a ReferenceError if outputBlobContainerUri is \'' + badUri + '\'', function() {
+        var registry = new Registry({ host: 'host', sharedAccessSignature: 'sas' });
+        assert.throws(function(){
+          registry.exportDevicesToBlobByIdentity(badUri, false, function() {});
+        }, ReferenceError);
+        assert.throws(function () {
+          registry.exportDevicesToBlobByIdentity(badUri, true, function () {});
+        }, ReferenceError);
+      });
+    });
+
+    testErrorCallback('exportDevicesToBlobByIdentity', 'input', true);
+
+    /*Tests_SRS_NODE_IOTHUB_REGISTRY_16_032: [The `exportDeviceToBlob` method shall construct an HTTP request using information supplied by the caller, as follows:
+    ```
+    POST /jobs/create?api-version=<version> HTTP/1.1
+    Authorization: <config.sharedAccessSignature>
+    Content-Type: application/json; charset=utf-8
+    Request-Id: <guid>
+
+    {
+      "type": "export",
+      "outputBlobContainerUri": "<output container Uri given as parameter>",
+      "excludeKeysInExport": "<excludeKeys Boolean given as parameter>"
```

Comment:
>"excludeKeysInExport": "<excludeKeys Boolean given as parameter> [](start = 6, length = 64)

Here too.  I mean you could just delete this requirement, but, that would be disappointing.